python_scripts/functions.py

```diff
@@ -215,6 +216,11 @@ def convert_meta_to_goodtables_schema(meta):
     return gt_template
 
 
+def log_validation_result(log, table_resp):
+    for e in table_resp["errors"]:
+        log.error(e["msg"], extra={"context": "VALIDATION"})
+
+
 def validate_data(config):
 
     utc_ts = int(datetime.utcnow())
```

Comment:
This line doesn't work - `TypeError: int() argument must be a string, a bytes-like object or a number, not 'datetime.datetime'`. Should be `int(datetime.utcnow().timestamp())` I believe. Curious why that didn't flag before now.